Makefile

```diff
@@ -11,21 +11,21 @@ compile:
 
 build: project-config
 	cp \
-		$(PROJECT_DIR)/build/automation/etc/certificate/* \
-		$(PROJECT_DIR)/application/src/main/resources/certificate
+		$(PROJECT_DIR)build/automation/etc/certificate/* \
+		$(PROJECT_DIR)application/src/main/resources/certificate
```

Comment:
I don't understand why you needed to remove the first '/' after the PROJECT_DIR variable. This is in a few places